pkg/apis/pravega/v1alpha1/pravegacluster_types.go

```diff
@@ -103,6 +107,10 @@ type ClusterSpec struct {
 	//pravega-bookie-2.pravega-bookie-headless.default:3181
 	BookkeeperUri string `json:"bookkeeperUri"`
 
+	// VersionMap is the name of the configMap which contains the list of
+	// supported versions for the pravega cluster
+	VersionMap string `json:"versionMap"`
```

Comment:
This is really a upgrade_supported_versions map. Variable name should reflect that.